README.md

```diff
@@ -41,96 +72,186 @@ pod "KlaviyoSwift"
+* Individual's email address
+* Individual's phone number in [E.164 format](https://help.klaviyo.com/hc/en-us/articles/360046055671#h_01HE5ZYJEAHZKY6WZW7BAD36BG)
+
+These above Identifiers are persisted to local storage so that the SDK can keep track of the current user/profile for you when you make event requests or want to set a push token etc.
+
+Profile identifiers can be set all at once or individually. Either way, the SDK will group and batch API calls to improve performance.
+
+The following code demonstrates how to set profile identifiers:
 
 ```swift
-KlaviyoSDK().initialize(with: "YOUR_KLAVIYO_PUBLIC_API_KEY")
+// organization, title, image, location and additional properties (dictionary) can also be set using the below constructor
+let profile = Profile(email: "junior@blob.com",  firstName: "Blob",  lastName: "Jr.")
+KlaviyoSDK().set(profile: profile)
+
+// or setting individual properties
+KlaviyoSDK().set(profileAttribute: .firstName, value: "Blob")
+KlaviyoSDK().set(profileAttribute: .lastName, value: "Jr.")
 ```
 
-3. Begin tracking events anywhere within your application by calling the `create(event:)` method in the relevant location.
+### Reset Profile
+To start a new profile altogether (e.g. if a user logs out) either call `KlaviyoSDK().resetProfile()` to clear the currently tracked profile identifiers,
+or use `KlaviyoSDK().set(profile: profile)` to overwrite it with a new profile object.
 
 ```swift
-let event = Event(name: .AddedToCartMetric, properties: [
-    "Total Price": 10.99,
-    "Items": ["Hot Dog", "Fries", "Shake"]
-], identifiers: .init(email: "junior@blob.com"),
-profile: [
-    "$first_name": "Blob",
-    "$last_name": "Jr"
-], value: 10.99)
+// start a profile for Blob Jr.
+let profile = Profile(email: "junior@blob.com",  firstName: "Blob",  lastName: "Jr.")
+KlaviyoSDK().set(profile: profile)
 
-KlaviyoSDK().create(event: event)
+// stop tracking Blob Jr.
+KlaviyoSDK().resetProfile()
+
+// start a profile for Robin Hood
+let profile = Profile(email: "robin@hood.com",  firstName: "Robin",  lastName: "Hood")
+KlaviyoSDK().set(profile: profile)
 ```
+### Anonymous Tracking Notice
 
-### Arguments
+By default, Klaviyo will begin tracking unidentified users in your app when a push token is set, or if it receives an event to track.
+This means you will be able to track events or set a push notification token from users in your app without any user information provided.
+When an email or other primary identifier is provided, Klaviyo will merge the data from the anonymous user to a new identified user.
 
-The `create` method takes an event object as an argument. The event can be constructed with the following arguments:
+Prior to version 1.7.0, the Klaviyo SDK used the [Apple identifier for vendor (IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking.
```

Comment:
Wonder at what point we can remove this, we're on 3.0 now, referring back to 1.7 seems ooold

Reply:
Yeah you are right. I removed this - @ndurell let me know if that's okay?
CC: @mark-piana